src/components/axes/numericAxis.ts

```diff
@@ -1,4 +1,4 @@
-//<reference path="../../reference.ts" />
+///<reference path="../../reference.ts" />
```

Comment:
I am unsure how to tell the linter to look for the above.  The above seems very specific to Plottable